conformance_automated = "Fail"
errors_diff = """
Line 25: Expected 1 errors
Line 38: Expected 1 errors
Lines 10, 11: Expected error (tag 'DC1')
Line 56: Unexpected errors ["dataclasses_slots.py:56:1: error[unresolved-attribute] Type `<class 'DC5'>` has no attribute `__slots__`"]
Line 57: Unexpected errors ['dataclasses_slots.py:57:1: error[unresolved-attribute] Type `DC5` has no attribute `__slots__`']
"""
output = """
dataclasses_slots.py:56:1: error[unresolved-attribute] Type `<class 'DC5'>` has no attribute `__slots__`
dataclasses_slots.py:57:1: error[unresolved-attribute] Type `DC5` has no attribute `__slots__`
dataclasses_slots.py:66:1: error[unresolved-attribute] Type `<class 'DC6'>` has no attribute `__slots__`
dataclasses_slots.py:69:1: error[unresolved-attribute] Type `DC6` has no attribute `__slots__`
"""
